### ISSUE TITLE
error

### ISSUE THREAD

**Issue by von216** [rohan1198/Left-Atrial-Segmentation#1]
pred = pred[0, 0].to(dtype=torch.uint8).detach().cpu().numpy()

**Comment by rohan1198** [comment]
Hi there, could you include the output error message as well? Thanks!

**Comment by von216** [comment]
Hi, the error is because of 
iou, p, r, f1, dice = SegmentationMetrics(pred, val_label[0,0].cpu().numpy()).metrics()
val_label should be 3 dimention

**Comment by rohan1198** [comment]
Thanks. I'll add a fix for it sometime soon. Please feel free to send a pull request if you want :)
